lanebreach-api/app/services/sf311_case_service.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'soda/client'
+
+module Sf311CaseService
+  extend self
+
+  SODA_CREDENTIALS = {
+    domain: 'data.sfgov.org',
+    app_token: 'QPCu2zzyc3jV5UkGfpOrGnXi7'
+  }
+
+  CASE_DATASET_ID = 'ktji-gk7t'
+
+  # https://dev.socrata.com/docs/datatypes/floating_timestamp.html
+  SODA_FLOATING_TIMESTAMP_FORMAT ='%Y-%m-%dT%T'
+
+  def get_blocked_bike_lane_cases
+    get_cases(Sf311Case::SERVICE_SUBTYPES[:blocked_bike_lane])
+  end
+
+  def create_case(case_attributes)
+    if Rails.env.production?
+      # TODO: Ensure this call behaves as expected
+      client.post(CASE_DATASET_ID, [case_attributes])
+    else
+      Rails.logger.info("#{self.name}##{__method__} called successfully, actual case creation disabled in this environment (#{Rails.env})")
```

Comment:
I'm not aware of how to access it but Dale was mentioning that sf actually has a test / dev environment for their 311 case data to which you can make POSTs too. Might be worth looking into.

Reply:
Awesome, I'll chat with Dale about that.